scylla/src/transport/cluster.rs

```diff
@@ -343,6 +348,70 @@ impl ClusterData {
     pub fn get_nodes_info(&self) -> &Vec<Arc<Node>> {
         &self.all_nodes
     }
+
+    /// Compute token of a table partition key
+    pub fn compute_token(
+        &self,
+        keyspace: &str,
+        table: &str,
+        partition_key: impl ValueList,
+    ) -> Result<Token, Box<dyn std::error::Error>> {
```

Comment:
I've hesitated to create a new Error for that, as it combines `SerializeValuesError` and `PartitionKeyError`. Can I have your opinion about it?

Reply:
perhaps it could reuse the generic part of SerializeValuesError (SerializeValuesError::ParseError), to wrap the partition key error part? If it's for any reason not viable, I'm also not against creating a separate error type.

Reply:
I'm not sure about using `ParseError`. I've also thought about using `BadQuery`, because it wraps both errors; maybe it's too much generic, but I finally like it. If you don't mind, I will use it.

Reply:
I'm fine with picking an error generic enough for both cases, go ahead

Reply:
done